static/scss/partials/landing/a.scss

```
@@ -167,7 +174,7 @@
 .c-gray-bg {
     background: linear-gradient(180deg, $color-light-gray-20 0%, $color-light-gray-10 3%);
     height: auto;
-    padding: $layout-md 0;
+    padding-top: $layout-md 0 $layout-xs 0;
```

There are four values here, but also a change to `padding-top`. If this works now, presumably it should be just:

```suggestion
    padding-top: $layout-md;
```

?

---

Nice call out - this was an error on my part, it's supposed to just be `padding-top` with one value.